docs/source/conf.py

```diff
@@ -62,16 +63,16 @@
 
 # General information about the project.
 project = u'Cartographer ROS'
-copyright = u'2016 The Cartographer Authors'
+copyright = u'{year} The Cartographer Authors'.format(year=datetime.now().year)
```

Comment:
no need to change this, the oldest content is from 2016, so fine as is

Reply:
Would be consistent with: https://github.com/googlecartographer/cartographer/pull/1268